opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/MinionFeatureKarafIT.java

```diff
@@ -75,4 +75,10 @@ public void testInstallFeatureOpennmsDiscoveryDistPollerDaoMinion() {
 		installFeature("opennms-discovery-distPollerDaoMinion");
 		System.out.println(executeCommand("features:list -i"));
 	}
+	
+	@Test
```

Comment:
Please add ITs for the blueprint features as well. If the tests do not work (because of feature interdependencies or permissions to open the trap port) then mark the failing test with an @Ignore annotation that explains why they don't work, for example:
```
@Ignore("This doesn't work because we cannot open the trap port")
```